### PR TITLE
docs: missing steps in first-app-lesson-11.md

### PR DESCRIPTION
**Pull request by grrd01**
docs(docs-infra): missing steps in first-app-lesson-11.md

add 2 missing steps in Step 1 - Create a new service for your app 
Before adding the anchor tag to the template, an import statement needs to be added and the imports of the component need to be expanded. The app will not work, if you skip this. Since this is a beginners-tutorial, I think those two steps need to be mentioned.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
In https://angular.io/tutorial/first-app/first-app-lesson-11, an anchor tag is added to the template of src/app/housing-location/housing-location.component.ts. Currently, there is no import statemend added and the imports are not expanded. The app crashes in that state.

Issue Number: N/A

## What is the new behavior?
With the two added steps, the import-statement is present and the imports contain  RouterModule, and the app runs as desired.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by JeanMeche** [comment]
Hi, can you reword your commit ? There is no need for a scope when doing do changes.  (`doc-infra` is meant for changes in the Angular.io app, not its content).

File: aio/content/tutorial/first-app/first-app-lesson-11.md [hunk 1]
```diff
@@ -25,13 +25,22 @@ In lesson 10, you added a second route to `src/app/routes.ts` which includes a s
...
-
-    For the `routerLink` to work in the template, add a file level import of `RouterLink` and `RouterOutlet` from '@angular/router', then update the component `imports` array to include both `RouterLink` and `RouterOutlet`.
+1.  In `src/app/housing-location/housing-location.component.ts`, update the component to use routing:
+    1.  Add a file level import for `RoutingModule`:
+
+        <code-example header="Import RouterModule in src/app/housing-location/housing-location.component.ts" path="first-app-lesson-11/src/app/housing-location/housing-location.component.ts" region="import-router-module"></code-example>
+    
+    1.  Add `RouterModule` to the `@Component` metadata imports
+
+        <code-example header="Import RouterModule in src/app/housing-location/housing-location.component.ts" path="first-app-lesson-11/src/app/housing-location/housing-location.component.ts" region="import-router-module-deco"></code-example>
```
Comment (JeanMeche) [inline aio/content/tutorial/first-app/first-app-lesson-11.md:35]: This doc region does not exist in the lesson-11. 

https://github.com/angular/angular/blob/05657cfde4514671570a674a4be83989729e2740/aio/content/examples/first-app-lesson-11/src/app/housing-location/housing-location.component.ts#L6-L24

**Review by JeanMeche — CHANGES_REQUESTED** [review]

File: aio/content/tutorial/first-app/first-app-lesson-11.md (hunk 1 above)
Comment (grrd01) [reply aio/content/tutorial/first-app/first-app-lesson-11.md:35]: Hi @JeanMeche  I've added another commit with the missing doc regions  now added. Hope this is ok now.

**Comment by JeanMeche** [comment]
The change is looking good thank you for improving the docs. One last thing, could you squash both commits ? Thank you!

**Comment by grrd01** [comment]
> The change is looking good, thank you for improving the docs! One last thing, could you squash both commits ? Thank you!

Hi @JeanMeche ! I've tried to squash the two commits into one, but to be honest, I'm reaching the limits of my git knowhow here. So - not sure if this worked the way you expected. Otherwise I'll need your help on this.

**Comment by JeanMeche** [comment]
Look at interactive rebase. (`git rebase -i`). It will allow you to squash (merge) commits, drop (delete) commit, etc. 

It's a very usefull and powerfull tool that might also help you in the future, it's worth taking time learning it 👍

**Comment by MarkTechson** [comment]
@grrd01 Were you able to figure out what you need to finish up this PR?

**Comment by grrd01** [comment]
I‘ve tried it twice, obviously without success. ThenI‘ve given up. Am 22.08.2023 um 00:01 schrieb Mark Thompson ***@***.***>:﻿
@grrd01 Were you able to figure out what you need to finish up this PR?

**Comment by jessicajaniuk** [comment]
@grrd01 In order to rebase and get the commits squashed and pushed to the repository, you need to follow a few steps:

1. Check out `main`, fetch upstream so that you're as up to date as possible
2. Check out your `patch-2` branch
3. run `git rebase main -i`
4. When you get to the screen that has all the commits, either fixup or squash them into the one commit
5. Once that rebase process has finished, push the changes with a force push: `git push [remote name here] patch-2 --force`

That should update the branch for your PR and eliminate the extra commits. During that process, if you need to reword a commit message, you can do that then.

After that, you'll need to sign the CLA. Then we can re-review and go from there.

**Comment by JeanMeche** [comment]
Without any feedback, I'll close this. Feel free to reopen if you'd like to land this change.

**Pull request closed without merging**